src/components/form.js

```diff
@@ -62,7 +62,7 @@ export default class Form extends Component {
               id="email"
               />
           </div>
-          <input className="cmp-form__button" type="submit" />
+          <input className="cmp-form__button" type="submit" value="Submit" />
```

Comment:
I didn’t pick up this before. Interesting we’re using an input type of submit instead of a `button` element. Would you mind switching to the button element?

Reply:
Sure thing. The [React docs](https://reactjs.org/docs/forms.html) use an `input` for some reason, but it seems to work with the `button` element as well.